Makefile

```diff
@@ -398,6 +398,15 @@ clippy_cuda_backend: install_rs_check_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" clippy --all-targets \
 		-p tfhe-cuda-backend -- --no-deps -D warnings
 
+.PHONY: check_rust_bindings_change # Check rust bindings are up to date for tfhe-cuda-backend
+check_rust_bindings_change: 
+	cargo build --release -p tfhe-cuda-backend && \
+		git diff --quiet HEAD -- backends/tfhe-cuda-backend/src/bindings.rs || { \
+		if [ -s backends/tfhe-cuda-backend/src/bindings.rs ]; then \
```

Comment:
I checked and `-s` apparently means " 
![image](https://github.com/user-attachments/assets/ec910998-7a92-410d-9df9-c0949783e74b)

maybe just echoing the message + having the exit code being non zero is enough ?

```bash
git diff --quiet HEAD -- backends/tfhe-cuda-backend/src/bindings.rs || ( echo "Generated bindings have changed! Please run 'git add backends/tfhe-cuda-backend/src/bindings.rs' and commit the changes." && exit 1 ) 
```
maybe ?

having a check first to see if the file exists with a -e and a dedicated error message to be a bit hardened vs edge cases we may not think of now ?

Reply:
Yes you're right, this is leftover of me trying to avoid getting an error even though the change on bindings was empty.